pyk/src/pyk/utils.py

```diff
@@ -159,7 +163,20 @@ def shorten_hashes(value: Any, leftChars=6, rightChars=6) -> Any:
     return result
 
 
+def deconstruct_short_hash(h: str) -> Tuple[str, str]:
+    x = h.lower()
+    if is_hash(x):
+        return (x, x)
```

Comment:
Shouldn't it be:

```suggestion
        return (x, '')
```

Also, I think we should support short hashes without the separator (e.g. `'abcd'`), interpreted as the hash prefix (e.g. `'abcd..'`).

Reply:
> Shouldn't it be: `(x, '')`

No, the first argument is what we want the prefix, and the second is what we want the suffix to match.
In the case when the full hash is supplied, but the left and the right must match. I guess it doesn't make a difference unless we're comparing with a shorthash thought.

> Also, I think we should support short hashes without the separator (e.g. 'abcd'), interpreted as the hash prefix (e.g. 'abcd..').

I'm worried about aliases also parsing as hashes though

Reply:
Good point. But this problem is already present, consider the case when someone provides an alias that is also a long hash. So maybe we should syntactically differentiate hashes from aliases by having a prefix character for one of them (e.g. `@somealias`, `#abcd`, etc).